com.reprezen.swagedit/src/com/reprezen/swagedit/assist/SwaggerContentAssistProcessor.java

```diff
@@ -57,28 +62,71 @@
 	private final JsonReferenceProposalProvider referenceProposalProvider = new JsonReferenceProposalProvider();
 	private final ContentAssistant contentAssistant;
 
+	/**
+	 * This string contains the pointer that helps us locate 
+	 * the current position of the cursor inside the document. 
+	 * 
+	 */
 	private String currentPath = null;
+
+	/**
+	 * Current position in the list of proposals.
+	 * Use for JSON Reference proposals.
+	 * 
+	 * @see JsonReferenceProposalProvider.Scope
+	 */
 	private int cyclePosition = 0;
+
+	/**
+	 * True if the proposal is activated on a JSON reference.  
+	 */
 	private boolean isRefCompletion = false;
 
-	private String[] textMessages = new String[] {
-			"Press 'Ctrl+Space' to show all schemas in the project.",
-			"Press 'Ctrl+Space' to show all schemas in the workspace.",
-			"Press 'Ctrl+Space' to show schemas in the current file."
-	};
+	private String[] textMessages;
 
 	public SwaggerContentAssistProcessor() {
 		this(null);
 	}
 
 	public SwaggerContentAssistProcessor(ContentAssistant ca) {
 		this.contentAssistant = ca;
+		this.textMessages = initTextMessages();
+	}
+
+	protected String[] initTextMessages() {
+		IBindingService bindingService = (IBindingService) 
+				PlatformUI.getWorkbench().getAdapter(IBindingService.class);
+
+		String bindingKey = bindingService
+				.getBestActiveBindingFormattedFor(IWorkbenchCommandConstants.EDIT_CONTENT_ASSIST);
+
+		String context;
+		switch(ContextType.get(currentPath)) {
+			case PATH_ITEM:
+				context = "path item";
+				break;
+			case PATH_PARAMETER:
+				context = "parameter";
+				break;
+			case PATH_RESPONSE:
+				context = "response";
+				break;
+			default:
+				context = "schema";
+				break;
+		}
+
+		return new String[] {
+				String.format(Messages.content_assist_proposal_project, bindingKey, context),
+				String.format(Messages.content_assist_proposal_workspace, bindingKey, context),
+				String.format(Messages.content_assist_proposal_local, bindingKey, context)
```

Comment:
And a screenshot:
<img width="395" alt="screen shot 2016-07-14 at 11 51 52 am" src="https://cloud.githubusercontent.com/assets/644582/16846203/7d6955f6-49b9-11e6-8b41-7b3dfbef6481.png">